test/e2e/singlecluster/pod_test.go

```diff
@@ -489,27 +490,28 @@ var _ = ginkgo.Describe("Pod groups", func() {
 				})
 			})
 
-			replacementPods := make(map[types.NamespacedName]types.NamespacedName, len(defaultPriorityGroup))
-			ginkgo.By("Create replacement pods as soon as the default-priority pods are Failed", func() {
-				gomega.Eventually(func(g gomega.Gomega) int {
+			ginkgo.By("Wait for default-priority pods are Failed", func() {
+				gomega.Eventually(func(g gomega.Gomega) {
 					for _, origPod := range defaultPriorityGroup {
-						origKey := client.ObjectKeyFromObject(origPod)
-						if _, found := replacementPods[origKey]; !found {
-							var p corev1.Pod
-							g.Expect(k8sClient.Get(ctx, origKey, &p)).To(gomega.Succeed())
-							if p.Status.Phase == corev1.PodFailed {
-								rep := origPod.DeepCopy()
-								// For replacement pods use args that let it complete fast.
-								rep.Name = "replacement-for-" + rep.Name
-								rep.Spec.Containers[0].Args = []string{"1ms"}
-								g.Expect(k8sClient.Create(ctx, rep)).To(gomega.Succeed())
-								replacementPods[origKey] = client.ObjectKeyFromObject(rep)
-							}
-						}
+						var p corev1.Pod
+						g.Expect(k8sClient.Get(ctx, client.ObjectKeyFromObject(origPod), &p)).To(gomega.Succeed())
+						g.Expect(p.Status.Phase).To(gomega.Equal(corev1.PodFailed))
 					}
-					return len(replacementPods)
-				}, util.Timeout, util.Interval).Should(gomega.Equal(len(defaultPriorityGroup)))
+				}, util.LongTimeout, util.Interval).Should(gomega.Succeed())
```

Comment:
Here is the same situation as we have on the [discussion](https://github.com/kubernetes-sigs/kueue/pull/2733#discussion_r1698415796). But on this case we can't wait for the pod to be Running to makes it stable because only one pod can be Running. 

cc @mimowo

Reply:
I see, thanks for the comment. Using LongTimeout is ok then.